docs/user_guide.md

```diff
@@ -85,9 +13,9 @@ You can use your tennis club account to log into the Courtgrabber program.
 You see a `Login Successful` message.
 
 *Figure 2: Logging into Courtgrabber.*
```

Comment:
Should this become Figure 1 now?

Reply:
If the docs were more extensive, I agree it would make sense to begin each page's figure numbering with 1. But since there are relatively few images, and the docs themselves are not lengthy, I recommend giving each figure a unique number so they can be referred to without confusion. Happy to discuss further, though.

Reply:
This is more your area of expertise than mine, so I'm happy to go with your recommendation.  Thanks.